Recruit-Training/Basic-Recruit-Training/person/Person.h

```diff
@@ -25,7 +25,6 @@ class Person
 private:
     std::string name_;
     int* age_; // age_ should be dynamically allocated
-
```

Comment:
As mentioned in the instructions, there shouldn't be any modification of `Person.h`

Reply:
Reverted the change